template-evidence.php

```diff
@@ -18,23 +33,56 @@
 				</header>
 
 				<div class="section-wrapper">
-					<section class="row halves gutter pad-ends">
+					<section class="row halves gutter pad-ends intro">
 						<div class="column one">
 							<?php the_content(); ?>
 						</div>
 						<div class="column two">
+							<div id="story-filters">
+								<div id="filter-options">
+									<p>View <span><select>
+										<option value="">All</option>
+										<?php
+										if ( ! empty( $terms ) ) {
+											foreach ( $terms as $term_option ) {
+												$name = explode( ', Academic', $term_option->name );
+												?>
+												<option value="<?php echo esc_attr( $term_option->slug ); ?>"<?php selected( $category, $term_option->slug ); ?>><?php echo esc_html( $name[0] ); ?></option>
+												<?php
+											}
+										}
+										?>
+									</select></span> stories</p>
+								</div>
+							</div>
 						</div>
 					</section>
 				</div>
 
+				<section class="row halves gutter pad-bottom topic-title">
+					<header>
+						<h3><?php echo esc_html( $heading[0] ); ?></h3>
+					</header>
+				</section>
+
 				<?php
 				$stories_query_args = array(
 					'post_type' => 'drive_story',
-					'posts_per_page' => -1,
```

Comment:
We should probably still keep an explicit default of 10 or something here so that the "posts per page" setting in WP doesn't accidentally affect it.